src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -11,7 +11,7 @@ class CrudField {
             this.name = name;
 
             // get the input/textarea/select that has that field name
-            this.$input = $('input[name="'+this.name+'"], textarea[name="'+this.name+'"], select[name="'+this.name+'"], select[name="'+this.name+'[]"]').first();
+            this.$input = $(`input[name="${this.name}"], textarea[name="${this.name}"], select[name="${this.name}"], select[name="${this.name}[]"]`).first();
```

Comment:
I trust you've already solved this, and dot names can be used with this. So I'm going to merge this as-is.

But I'm just wondering. **Why is this any better than before?** Does it cover a scenario I'm not aware of? I'm not saying we should change it, but for the future, I want us to keep this in mind. 

The way I see it - this change only makes the code more difficult to read. Why use "`" and "${}" when "'" will do the trick just fine?

Reply:
@tabacitu ouch 😅

**In my opinion**, it is easier to read, because it ends with the mess of `"'+` `+'"`, `'"'`, `"'"`.
It's also easier to spot a missing `"` or `'`.

```js
'[name="'+name+']'
```

```js
`[name="${name}]`
```

Isn't it easier to spot the missing `"` in the template strings?

By using template strings you no longer need to worry about quotes or apostrophe, and switch to quotes because you have an apostrophe in the string `'isn't'`, template strings just work 🤷‍♂️

Sorry, I've been spreading template strings along the backpack code without asking, never thought about the chance anyone would not like template strings 😅

But let me know your thoughs @tabacitu.

Reply:
> Isn't it easier to spot the missing " in the template strings?

Not really, no 😅 I think what helps in both cases is syntax highlighting.

> By using template strings you no longer need to worry about quotes or apostrophe, and switch to quotes because you have an apostrophe in the string 'isn't', template strings just work 🤷‍♂️

Ok that's actually nice, you're right. Good argument.

> Sorry, I've been spreading template strings along the backpack code without asking, never thought about the chance anyone would not like template strings 😅

Yeah no worries. If I had a definite opinion, you would have heard it by now. I guess... I'm not a big fan, but not against them either. Definitely not worth refactoring to template strings in unrelated PRs though - I think it distracts from the functionality at hand, and gives the reviewer more things to test, without any additional benefit.

I don't know 🤷‍♂️ We'll see. We'll revisit 😀